modules/diskq/qdisk.c

```diff
@@ -560,18 +601,12 @@ _load_state(QDisk *self, GQueue *qout, GQueue *qbacklog, GQueue *qoverflow)
 
   if (!self->options->read_only)
     {
-      end_ofs = qout_ofs;
```

Comment:
Just a note: so the bug was in this line: if qout_ofs was zero, the minimum calculation logic did nothing, and in the end syslog wouldn't truncate.

Reply:
Yes.